torch/_inductor/metrics.py

```diff
@@ -62,6 +62,63 @@ def reset():
     disable_cpp_wrapper = 0
 
 
+@dataclass
+class CachedMetricsDeltas:
```

Comment:
@eellison this is what I mentioned. WDYT?

Reply:
Looks good, maybe in the future we can migrate file an issue to migrate these to the counters mechanism and centralize under that.

Not needed for this pr